hack/cluster-push.sh

```diff
@@ -34,10 +34,12 @@ if [[ "${podman:-}" =~ "docker" ]]; then
 else
   imgstorage="containers-storage:"
 fi
-skopeo copy --dest-tls-verify=false --dest-creds unused:${secret} "${imgstorage}${LOCAL_IMGNAME}" "docker://${registry}/${REMOTE_IMGNAME}"
```

Comment:
Is this just "we're substituting podman for skopeo because podman can do it just as well", or is there a specific issue we're avoiding here? (e.g. "skopeo does X wrong", "skopeo is missing in some environments", etc)